### DISCUSSION TITLE
Runoff poll for the name of a new IPFS implementation in JS

### DISCUSSION THREAD

**Discussion by TinyTb** [ipfs/ipfs#496] [Poll] [answered]
**Update:** The name [Helia](https://github.com/ipfs/helia) was selected. 🎉

<hr>

Per [State of IPFS in JS (blog post)](https://blog.ipfs.tech/state-of-ipfs-in-js/) and https://github.com/ipfs/pomegranate/issues/3, we're seeking a name for a new implementation of IPFS in JavaScript.

In the original request for name ideas, there were 19 suggested names. IP Stewards carefully reviewed and discussed all submissions and has selected two finalists, based on the consideration of many criteria: availability on npm; attempting to avoid names that clash with other projects; names that have potential problems with hearing and pronunciation; etc.

The two finalists are: **Helia** and **Sfero**.

*Proposed name:* 

# helia
To vote for `helia`, react to this post with 🎉 . You can vote for both names.

*Reason it’s great:* 

Helia is the latin spelling of Ἡλιη (Helie or hee-lee phonetically, emphasis on the first syllable), in Greek mythology one of the Heliades - the daughters of the sun god Helios.  When their brother Phaethon died trying to drive the sun chariot across the sky, their tears of mourning fell to earth as amber, which is yellow (sort of), and so is JavaScript.  They were then turned into poplar trees and, well, JavaScript is quite popular.

*How it reads in multiple sentences:* 
1. Add a directory with 'helia add -r foo'
2. You can fetch a file with 'helia get'
3. All the helia nodes are having trouble exchanging data with the kubo nodes

<hr>

*Proposed name:* 

# sfero
To vote for `sfero`, react to this post with 🚀  . You can vote for both names.

*Reason it’s great:* 

sfero is "sphere" in Esperanto, as kubo is "cube", to stay in volumes semantics. (maybe the python version will be pyramid ? :P )

Assuming no major conflicts, we could alias the binary to "sf" (in package.json) to save a few keystrokes:

*How it reads in multiple sentences:* 
1. Add a directory with 'sf add -r foo'
2. You can fetch a file with 'sf get'
3. All the sfero nodes are having trouble exchanging data with the kubo nodes

# When?

Per https://github.com/ipfs/pomegranate/issues/3, we announced the vote during IPFS Camp 2022. This runoff poll is now active. **This poll will end on 2022-12-08.**

We are taking this poll in order to collect opinions from the community, but IP Stewards reserves the right to select any name for this project, for any reason.

# How to vote

For maximum simplicity, the vote is taken by emoji reactions to this post. If you like both names, **you can vote in favor of both names**!

🎉  = Helia
🚀  = Sfero

**Comment by devingfx** [comment]
Science against religion... again !! :disappointed:

**Reply by daviddias** [reply]
`Helia` is a beautiful name 😍

**Comment by TinyTb** [comment, marked as answer]
Helia is the winning name, with 18 votes vs. 10. On 2022-12-09, IP Stewards reviewed the results and finalized the choice. 🎉
